.clj-kondo/config.edn

```diff
@@ -83,7 +83,9 @@
     schema.core/either                                                           {:namespaces ["metabase.*"]}}}
 
   :discouraged-var
-  {clojure.string/lower-case {:message "Use metabase.util/lower-case-en instead of clojure.string/lower-case"}
+  {clojure.core/println      {:message "Use clojure.tools.logging instead of clojure.core/println"}
+   clojure.core/printf       {:message "Use clojure.tools.logging instead of clojure.core/printf"}
```

Comment:
Is it worth adding `clojure.core/print` here?

Reply:
Yeah, maybe prn as well.

Reply:
Added print, prn, and pr to the config. Will fix next, but pushing this will prevent auto-merge.

Reply:
Added all the prints....